include/learning/InitializeProfile.h

```diff
@@ -0,0 +1,126 @@
+#ifndef InitializeProfile_H
+#define InitializeProfile_H
+
+#include "../app.h"
+#include "../types/AlternativesPerformance.h"
+#include "../types/Categories.h"
+#include "../types/MRSortModel.h"
+#include "../types/Profiles.h"
+
+class ProfileInitializer {
+public:
+  /**
+   * ProfileInitializer standard constructor.
+   *
+   * @param altPerfs AlternativesPerformance objet that represents our dataset
+   * @param config config setup for logging
+   * @return ProfileInitializer object
+   */
+  ProfileInitializer(Config &config, AlternativesPerformance &altPerfs);
+
+  /**
+   * ProfileInitializer standard copy constructor.
+   *
+   * @param profInit ProfileInitializer object
+   * @param config config setup for logging
+   * @return ProfileInitializer object
+   */
+  ProfileInitializer(Config &config, const ProfileInitializer &profInit);
+
+  /**
+   * ProfileInitializer standard deconstructor
+   *
+   */
+  ~ProfileInitializer(){};
```

Comment:
nit: the implementation of the destructor should be in the cpp file:

in the .h file:
```
~ProfileInitializer();
```

in the .cpp file:
```
ProfileInitializer::~ProfileInitializer(){ }
```